packages/assertion-monitor/index.ts

```diff
@@ -0,0 +1,193 @@
+      indexed: true,
+    },
+    {
+      type: 'bytes32',
+      name: 'executionHash',
+      indexed: false,
+    },
+    {
+      type: 'tuple',
+      name: 'assertion',
+      components: [
+        {
+          type: 'tuple',
+          name: 'beforeState',
+          components: [
+            {
+              type: 'tuple',
+              name: 'globalState',
+              components: [
+                {
+                  type: 'bytes32[2]',
+                  name: 'bytes32Vals',
+                },
+                {
+                  type: 'uint64[2]',
+                  name: 'u64Vals',
+                },
+              ],
+            },
+            {
+              type: 'uint8',
+              name: 'machineStatus',
+            },
+          ],
+        },
+        {
+          type: 'tuple',
+          name: 'afterState',
+          components: [
+            {
+              type: 'tuple',
+              name: 'globalState',
+              components: [
+                {
+                  type: 'bytes32[2]',
+                  name: 'bytes32Vals',
+                },
+                {
+                  type: 'uint64[2]',
+                  name: 'u64Vals',
+                },
+              ],
+            },
+            {
+              type: 'uint8',
+              name: 'machineStatus',
+            },
+          ],
+        },
+        {
+          type: 'uint64',
+          name: 'numBlocks',
+        },
+      ],
+      indexed: false,
+    },
+    {
+      type: 'bytes32',
+      name: 'afterInboxBatchAcc',
+      indexed: false,
+    },
+    {
+      type: 'bytes32',
+      name: 'wasmModuleRoot',
+      indexed: false,
+    },
+    {
+      type: 'uint256',
+      name: 'inboxMaxCount',
+      indexed: false,
+    },
+  ],
+} as const
+
+const getLogsForLast7Days = async (
+  client: PublicClient,
+  address: `0x${string}` | `0x${string}`[],
+  abi: AbiEvent
+) => {
+  const latestBlockNumber = await client.getBlockNumber()
+  const oneWeekInSeconds = 7 * 24 * 60 * 60
+  const earliestBlock = await client.getBlock({
+    blockNumber: latestBlockNumber - BigInt(oneWeekInSeconds / 12),
+  }) // assuming average block time of 12 seconds
+
+  return await client.getLogs({
+    address,
+    event: nodeCreatedEventAbi,
+    fromBlock: earliestBlock.number,
+    toBlock: latestBlockNumber,
+  })
+}
+
+const monitorNodeCreatedEvents = async (childChainInformation: ChainInfo) => {
+  const parentChain = getChainFromId(childChainInformation.parentChainId)
+  const client = createPublicClient({
+    chain: parentChain,
+    transport: http(childChainInformation.parentRpcUrl),
+  }) as any
+
+  const logs = await getLogsForLast7Days(
+    client,
+    childChainInformation.ethBridge.rollup as `0x${string}`,
+    nodeCreatedEventAbi
+  )
+
+  if (!logs || logs.length === 0) {
+    const alertMessage = `No assertion events found on ${childChainInformation.name} in the last 7 days.`
+    console.error(alertMessage)
```

Comment:
I don't think we can directly infer that no-assertions will need to be directly alerted. We need to add an additional check if are there any batches being posted in the `ASSERTION_FOUND_TIME_WINDOW (name not final)`. ie. If the chain is inactive/batches are not being posted, we will not be having any assertions to begin with - currently leading to false positive alerts.

Context - https://offchainlabs.slack.com/archives/C048LPUQ2LA/p1722944019457749?thread_ts=1722917211.763949&cid=C048LPUQ2LA